src/Cms/Responder.php

```diff
@@ -431,4 +484,57 @@ public static function isPrivate(bool $usesAuth, array $usesCookies): bool
 
 		return false;
 	}
+
+	/**
+	 * Marks headers (or header parts) as request-dependent, so they
+	 * can be subtracted before caching a response snapshot
+	 *
+	 * @since 5.2.0
+	 */
+	public function markRequestDependentHeader(string $name, array|null $values = null): void
```

Comment:
`Todo`: Probably a leftover from the previous name:

```suggestion
	public function markVolatileHeader(string $name, array|null $values = null): void
```

Maybe the name could also be just `volatileHeader()` for consistency with `usesCookie()` etc.

Reply:
I agree. Yes, a leftover from the renaming.